src/output.h

```diff
@@ -203,14 +203,13 @@ int draw_item_info(const int iLeft, int iWidth, const int iTop, const int iHeigh
 char rand_char();
 long special_symbol (long sym);
 
-// TODO: move these elsewhere
-// string manipulations.
-std::string from_sentence_case (const std::string &kingston);
-
 std::string string_format(const char *pattern, ...);
 std::string vstring_format(const char *pattern, va_list argptr);
-std::string string_format(const std::string pattern, ...);
-std::string vstring_format(const std::string pattern, va_list argptr);
+std::string string_format(std::string pattern, ...);
+std::string vstring_format(std::string pattern, va_list argptr);
```

Comment:
Forgot to update the prototype to match the definition, I'm getting linker errors.